db/version_set.cc

```diff
@@ -1307,14 +1307,15 @@ VersionSet::UpdatePenalty(
         {
 
             // compute penalty for write throttle if too many Level-0 files accumulating
-            if (config::kL0_CompactionTrigger < v->files_[level].size())
+            if (config::kL0_SlowdownWritesTrigger < v->files_[level].size())
             {
                 // assume each overlapped file represents another pass at same key
                 //   and we are "close" on compaction backlog
                 if ( v->files_[level].size() < config::kL0_SlowdownWritesTrigger)
                 {
-                    value = (v->files_[level].size() - config::kL0_CompactionTrigger);
-                    count=0;
+                    // this code block will not execute due both "if"s using same values now
```

Comment:
I may change my mind and activate it.  The whole throttle may need another tuning based upon bugs found in Riak today (if the bugs are fixed).